docs/getting-started-on-the-raspberry-pi.md

```diff
@@ -0,0 +1,104 @@
+# Getting Started on the Raspberry Pi 3
+
+This guide assumes you are using the [Raspbian image]. It should work on other
+distributions as well, but some details may be different.
+
+We will compile the driver on the Raspberry Pi itself, of course you can also
+cross compile from your system. 
```

Comment:
Should I add a note about ssh and internet access on the Pi?

Reply:
I think it's fair to assume that the user set that up before; also see **https://github.com/winkj/raspberrypi-ess#introduction**

Reply:
Thanks. I changed it accordingly.